test/src/test/java/org/corfudb/CorfuTestParameters.java

```diff
@@ -87,8 +87,8 @@ public CorfuTestParameters(){
 
         // Timeouts
         TIMEOUT_VERY_SHORT = Duration.of(500, MILLIS);
-        TIMEOUT_SHORT = Duration.of(1, SECONDS);
-        TIMEOUT_NORMAL = Duration.of(10, SECONDS);
+        TIMEOUT_SHORT = Duration.of(5, SECONDS);
+        TIMEOUT_NORMAL = Duration.of(60, SECONDS);
         TIMEOUT_LONG = Duration.of(2, MINUTES);
```

Comment:
This should be part of a separate PR.

Reply:
Yeah this will be removed, it's temporary for debugging a flaky test, sorry for the confusion